src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -973,22 +974,26 @@ export default class GeFormRenderer extends LightningElement{
                 lastName: accountName
             }
         } else {
-            return {firstName, lastName};
+            return { firstName, lastName };
         }
     }
 
     get donorNames() {
-        return {
-            firstName: this.getFieldValueFromFormState(
-                DATA_IMPORT_CONTACT1_FIRSTNAME_FIELD
-            ),
-            lastName: this.getFieldValueFromFormState(
-                DATA_IMPORT_CONTACT1_LASTNAME_FIELD
-            ),
-            accountName: this.getFieldValueFromFormState(
-                DATA_IMPORT_ACCOUNT1_NAME
-            )
+        let donorNames = {};
+
+        const nameFields = {
```

Comment:
What do you think about this donorNames function returning an instance of `NameFields` that exposes methods for firstName(), lastName(), and accountName()? (each returning a value that is from either formState or cardholder state)

Reply:
Really like this idea. We would pass in both state objects in the `NameFields` constructor?